tests/basic/src/main/java/mil/emp3/dev_test_sdk/MainActivity.java

```diff
@@ -681,22 +673,22 @@ private void plotManyMilStd(int iMaxCount) {
         }
     }
 
-    private void processSymbolTable(java.util.Map<java.lang.String,UnitDef> oDefMap,
+    private void processSymbolTable(Map<java.lang.String,UnitDef> oDefMap,
             org.cmapi.primitives.GeoMilSymbol.SymbolStandard eStandard,
             int iMilStdVersion,
             int iMaxCount) {
-        java.util.Set<String> oSymbols = oDefMap.keySet();
+        Set<String> oSymbols = oDefMap.keySet();
         UnitDef oSymDef;
         String sSymbolCode;
         int iCount = 0;
-        java.util.List<IFeature> oFeatureList = new java.util.ArrayList<>();
+        List<IFeature> oFeatureList = new ArrayList<>();
 
         for (String sBasicSymbolCode: oSymbols) {
             //Log.d(TAG, "Symbol " + sBasicSymbolCode);
             if (SymbolUtilities.isWarfighting(sBasicSymbolCode)) {
                 oSymDef = oDefMap.get(sBasicSymbolCode);
 
-                java.util.List<IGeoPosition> oPosList = this.getPositions(1, 20000.0);
+                List<IGeoPosition> oPosList = this.getPositions(1, 20000.0);
```

Comment:
Is 20000.0 a magic number?  Should this be a named `const` or something similar?